tslint.json

```diff
@@ -1,7 +1,12 @@
 {
   "extends": ["tslint:latest", "tslint-react"],
   "rules": {
+    "arrow-parens": false,
     "jsx-no-multiline-js": false,
+    "object-literal-sort-keys": false,
+    "one-line": false,
+    "ordered-imports": false,
+    "trailing-comma": false,
```

Comment:
Not sure I needed all of these...just copied the `tslint` out of @kswenson's current branch. Some of these, like disabling `object-literal-sort-keys`, seem especially sensible...not being able to write `{firstName, middleName, lastName}` seems silly.